tests/automation/utilities/utils.ts

```diff
@@ -327,22 +327,52 @@ export async function hasElementBeenDeleted(
 export async function hasTextMessageBeenDeleted(
   window: Page,
   text: string,
-  maxWait?: number,
-): Promise<boolean> {
-  try {
-    await waitForElement(
-      window,
-      'data-testid',
-      'control-message',
-      maxWait,
-      text,
-    );
-    return false; // Text message was found
-  } catch (e) {
-    return true; // Text message doesn't exist or wasn't found in time
-  }
+  maxWait: number = 30000,
+) {
+  let el: ElementHandle<SVGElement | HTMLElement> | undefined = undefined;
+
+  await doWhileWithMax(
+    15000,
+    500,
+    'waiting for text message to be deleted',
+    async () => {
+      try {
+        el = await waitForElement(
+          window,
+          'data-testid',
+          'control-message',
+          maxWait,
```

Comment:
you allow 30000ms as maxWait here, which means that if the element is not visible (and you want it to not be) it will take 30s to throw as I understand it?
You want this to be 5s maybe, or even less?

Reply:
I'm not sure why its set to 30000ms, i'll change it to 5000ms